README.md

```diff
@@ -171,3 +302,19 @@ $ cat > templates/xml/my_mentoring_scenario.xml
 
 Restart the workbench to take the new scenarios into account.
 
+Upgrading from Version 1
+------------------------
+
+To upgrade a course from the earlier version of this XBlock, run the following
+command on a system with edx-platform and xblock-mentoring installed:
+
+```bash
+$ SERVICE_VARIANT=cms DJANGO_SETTINGS_MODULE="cms.envs.devstack" python -m mentoring.v1.upgrade "Org/Course/Run"
+```
+Where "Org/Course/Run" is replaced with the ID of the course to upgrade.
+
+License
+-------
+
+The Mentoring XBlock is available under the GNU Affero General
+Public License (AGPLv3).
```

Comment:
Can `doc/img/edit-view.png` be removed, since there's no longer XML editing? I don't see this in the README anymore.